### ISSUE TITLE
Stream title always changed to "Stream"

### ISSUE THREAD

**Issue by vale-max** [bluenviron/mediamtx#979]
## Which version are you using?

v0.19.0

## Which operating system are you using?

- [ ] Linux amd64 standard
- [ ] Linux amd64 Docker
- [x] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [ ] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

Description
When streaming from ffmpeg using Title metadata the title gets always changed to "Stream"

The conversation between ffmpeg and the server is:
```
OPTIONS rtsp://localhost:8554/virgin_radio RTSP/1.0
CSeq: 1
User-Agent: Lavf59.16.100

RTSP/1.0 200 OK
CSeq: 1
Public: DESCRIBE, ANNOUNCE, SETUP, PLAY, RECORD, PAUSE, GET_PARAMETER, TEARDOWN
Server: gortsplib

ANNOUNCE rtsp://localhost:8554/virgin_radio RTSP/1.0
Content-Type: application/sdp
CSeq: 2
User-Agent: Lavf59.16.100
Content-Length: 299

v=0
o=- 0 0 IN IP4 127.0.0.1
s=Dab radio
c=IN IP4 127.0.0.1
t=0 0
a=tool:libavformat 59.16.100
m=audio 0 RTP/AVP 96
b=AS:160
a=rtpmap:96 MPEG4-GENERIC/48000/2
a=fmtp:96 profile-level-id=1;mode=AAC-hbr;sizelength=13;indexlength=3;indexdeltalength=3; config=119056E500
a=control:streamid=0

```
Then the log between the server and mpv:
```
OPTIONS rtsp://raudio.lan:8554/virgin_radio RTSP/1.0
CSeq: 1
User-Agent: libmpv

RTSP/1.0 200 OK
CSeq: 1
Public: DESCRIBE, ANNOUNCE, SETUP, PLAY, RECORD, PAUSE, GET_PARAMETER, TEARDOWN
Server: gortsplib

DESCRIBE rtsp://raudio.lan:8554/virgin_radio RTSP/1.0
Accept: application/sdp
CSeq: 2
User-Agent: libmpv

RTSP/1.0 200 OK
CSeq: 2
Content-Base: rtsp://raudio.lan:8554/virgin_radio/
Content-Length: 257
Content-Type: application/sdp
Server: gortsplib

v=0
o=- 0 0 IN IP4 127.0.0.1
s=Stream
c=IN IP4 0.0.0.0
t=0 0
m=audio 0 RTP/AVP 96
a=rtpmap:96 mpeg4-generic/48000/2
a=fmtp:96 profile-level-id=1; mode=AAC-hbr; sizelength=13; indexlength=3; indexdeltalength=3; config=119056e500
a=control:trackID=0
```

## Describe how to replicate the issue

1. start the server
2. publish with:
`ffmpeg -re -stream_loop -1 -ar 48000 -ac 2 -f s16le  -i $MYPIPE -vn -c:a aac -b:a 160k -metadata title="Dab radio" -f rtsp  rtsp://localhost:$3/$4 >/dev/null 2>&1 &
`
3. read with mpd or mpv

## Did you attach the server logs?

no

## Did you attach a network dump?

no

**Comment by robotiste** [comment]
are we able to transfer the metadata in any of the protocol?

**Comment by aler9** [comment]
The issue is that this kind of initial metadata is really protocol specific - we can surely support it, but we will never be able to entirely convert it from a protocol to another.

**Comment by aler9** [comment]
Fixed by https://github.com/bluenviron/gortsplib/pull/371

**Comment by aler9** [comment]
fixed in MediaMTX v1.0.1
